coilsnake/ui/cli.py

```diff
@@ -45,6 +45,26 @@ def main():
     patch_rom_parser.add_argument("patch")
     patch_rom_parser.add_argument("headered")
     patch_rom_parser.set_defaults(func=_patchrom)
+    
+    expand32_parser = subparsers.add_parser("expand32", help="Expand a ROM's size to 32 MBits (4MB)")
```

Comment:
would it be better if there was a single "expand" command that takes in the size as an additional parameter (32 or 48)?